### PR TITLE
MediaItem to tailwindcss

### LINKED ISSUE #8232: Migration `MediaItem` to `tailwindcss`
### Is your feature request related to a problem?

Context is [here](https://github.com/kodadot/nft-gallery/issues/8222) 

### Describe the solution you would like

--

### Describe alternatives you have considered

--

### Screenshots

_No response_

### Are there opened related issues?

https://github.com/kodadot/nft-gallery/issues/8222

### PR DESCRIPTION
**Pull request by shashkovdanil**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #8232  
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [x] I've tested it at </ksm/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP&usdamount=0)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 3374f99</samp>

This pull request refactors several components in the `libs/ui/src/components/MediaItem` folder to use TailwindCSS instead of custom SCSS styles, to improve the UI design and performance of the NFT gallery. It also fixes some type errors, removes unused code, and updates the global styles and other components accordingly.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 3374f99</samp>

> _The `MediaItem` component got a makeover_
> _With TailwindCSS to make it look bolder_
> _No more custom SCSS_
> _Or unused props to stress_
> _The code is now cleaner and easier to go over_

### CODE REVIEW

**Comment by roiLeo** [comment]
> hmm tests failed after merge main branch. It looks like we have broken tests in the main branch

ref #8265

**Comment by daiagi** [comment]
colides with #8264 doesn't it?

**Review by roiLeo — COMMENTED** [review]
should we merge #8274 first since I'm moving `MediaItem` inside koda repo?

**Review by preschian — APPROVED** [review]

File: libs/ui/src/components/MediaItem/MediaItem.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
```
Comment (roiLeo) [inline libs/ui/src/components/MediaItem/MediaItem.vue:1]: this file is used only for preview ui, we use the one in `components/basic/MediaItem`

**Review by roiLeo — COMMENTED** [review]
otherwise lgtm

**Review by roiLeo — APPROVED** [review]

**Pull request merged**